en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md

```diff
@@ -0,0 +1,285 @@
+# AmazonSQS Connector Example
+
+The WSO2 Amazon SQS connector allows you to access the exposed Amazon SQS API from an integration sequence.
+
+## What you'll build
+
+This example explains how to use Amazon SQS Connector to create a queue in the Amazon SQS, send a message to the queue, forward it to Simple Stock Quote Service Backend, and send the response to the user. 
+
+It has a single HTTP API resource, which is `sendToQueue`. 
+
+<img src="{{base_path}}/assets/img/integrate/connectors/amazonsqs-connector.jpg" title="AmazonSQS-Connector" width="800" alt="AmazonSQS-Connector"/>
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Set up the environment 
+
+1. Follow the steps mentioned in the [Setting up the Amazon S3 Environment]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document to create an Amazon account and obtain access key ID and secret access key. Keep them saved to use in the next steps.
```

Comment:
This link doesn't contain any details about Amazon account creation or obtaining access keys. Instead shall we point to this doc? https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-setting-up.html